portal/academy/models.py

```diff
@@ -11,11 +9,17 @@ class Specialization(models.Model):
     code = models.CharField(max_length=255, primary_key=True)
     name = models.CharField(max_length=255)
     description = models.TextField(blank=True)
-    created = models.DateTimeField(auto_now_add=True)
+    created = models.DateTimeField(editable=False)
```

Comment:
Is this some change that ended up not being used?

Reply:
the `auto_now_add` was not working properly. I searched a bit and found some recommendations to use `editable=False` + the custom save function)

Reply:
Reversed these changes  as discussed via slack 😉